### PR TITLE
Fee in harvesting

### PR DESCRIPTION
**Pull request by domakun**
add single dialog to set transaction fee and password

### CODE REVIEW

File: src/core/model/Listeners.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export class Listeners {
         })
 
         this.listener.status(address).subscribe((error: TransactionStatusError) => {
+          //@ts-ignore
```
Comment (decentraliser) [inline src/core/model/Listeners.ts]: why?

File: src/language/en-US.json [hunk 2]
```diff
@@ -927,6 +927,7 @@
   "Finish": "Finish",
   "Type_Mnemonic_Phrase": "Type Mnemonic Phrase",
   "Are_you_sure_to_unlink_this_account":"Are you sure to unlink this account?",
+  "remote_account_being_created": "Please click the 'Generate' button to generate a valid remote account",
```
Comment (decentraliser) [inline src/language/en-US.json]: "Please click the 'Generate' button to generate a remote account"

File: src/components/forms/delegated-dialog/proxy-setting/ProxySettingTs.ts [hunk 3]
```diff
@@ -1,129 +1,135 @@
...
-                LinkAction.Link,
-                this.wallet.networkType,
-                feeAmount
-            )
-        } catch (error) {
-            throw new Error(error)
-        }
-    }
-
-    submit() {
```
Comment (decentraliser) [inline src/components/forms/delegated-dialog/proxy-setting/ProxySettingTs.ts]: please avoid to lint the whole file in these case

When accunt is unlinked:
Click submit, don't enter your password and exit the password dialog, you can't click on link anymore

File: src/components/password-and-fee-dialog/PasswordAndFeeDialogTs.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    }
+  }
+
+  passwordValidated(value) {
+    this.$emit('passwordValidated', {
+      password:value,
+      feeAmount:this.feeAmount
+    })
+    this.show = false
+  }
```
Comment (decentraliser) [inline src/components/password-and-fee-dialog/PasswordAndFeeDialogTs.ts]: if you enter a wrong password, the dialog closes without alert

File: src/components/forms/delegated-dialog/proxy-setting/ProxySettingTs.ts [hunk 5]
```diff
@@ -1,129 +1,128 @@
...
+  }
 
-    get password() {
-        return this.activeAccount.temporaryLoginInfo.password
-    }
+  checkPasswordBeforeGenerate(password) {
+    if (!password) return
+    this.password = password
+    this.setRemotePublicKey(password)
+  }
```
Comment (evias) [inline src/components/forms/delegated-dialog/proxy-setting/ProxySettingTs.ts]: why is this not using standard password checks?

Comment (domakun) [reply src/components/forms/delegated-dialog/proxy-setting/ProxySettingTs.ts]: what is standard password checks? The password is from passwordCheck component.

File: src/language/en-US.json [hunk 6]
```diff
@@ -936,6 +937,12 @@
...
   "parent_namespace": "Parent namespace",
+  "generate": "Generate",
+  "Enter_your_password_to_create_a_remote_account": "Enter your password to create a remote account",
+  "Please_set_transaction_fee": "Please set transaction fee",
+  "Please_set_transaction_fee_and_password": "Please set transaction fee and password",
   "Input_namespace_name": "Enter a namespace name",
-  "confirm_information_before_add_child_wallet": "Enter your password to create a new child wallet"
+  "confirm_information_before_add_child_wallet": "Enter your password to create a new child wallet",
+  "loading": "loading...",
+  "Enter_your_password_to_create_a_link_transaction": "Enter your password to create a remote link transaction"
```
Comment (evias) [inline src/language/en-US.json]: `Enter your password to create an account link transaction`

Comment (domakun) [reply src/language/en-US.json]: fixed

**Pull request merged**